drafts/2019-11-06-1900011.adoc

```diff
@@ -157,6 +160,95 @@ MetadataBuilder mdb = Metadata.builder();
+        <file name="<path>/jaegerLib_0.34/gson-2.8.2.jar" />
+        <file name="<path>/jaegerLib_0.34/jaeger-client-0.34.0.jar" />
+        <file name="<path>/jaegerLib_0.34/jaeger-core-0.34.0.jar" />
+        <file name="<path>/jaegerLib_0.34/jaeger-thrift-0.34.0.jar" />
+        <file name="<path>/jaegerLib_0.34/jaeger-tracerresolver-0.34.0.jar" />
+        <file name="<path>/jaegerLib_0.34/libthrift-0.12.0.jar" />
+        <file name="<path>/jaegerLib_0.34/slf4j-api-1.7.25.jar" />
+        <file name="<path>/jaegerLib_0.34/slf4j-jdk14-1.7.25.jar" />
+        <file name="<path>/jaegerLib_0.34/opentracing-util-0.31.0.jar" />
+        <file name="<path>/jaegerLib_0.34/opentracing-noop-0.31.0.jar" />
+    </library>
+----
+
+`- define your appplication:`
+
+[source, xml]
+
+ <webApplication location="yourapp.war" contextRoot="/yourapp">
+        <!-- enable visibility to third party apis -->
+        <classloader commonLibraryRef="jaegerLib"
+            apiTypeVisibility="api,ibm-api,spec,stable,third-party" />
+    </webApplication>
+
+You can find out more about about Jaeger settings set up using environment variables by looking at link:https://github.com/jaegertracing/jaeger-client-java/blob/10c641f8df6316f1eac4d5b1715513275bcd724e/jaeger-core/README.md[jaeger-client-java]
+
+** For the `JAEGER_PASSWORD`, the password can be encoded using the `securityUtility command`
+** Depending on Jaeger's sampling settings `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`, Jaeger may not report every spans created by the applications.
+
+[#LAC]
+== Liberty Annotation Caching
+
+During application startup, application classes are scanned for class signature information and for annotations. Typically, all application classes are scanned, which is time consuming.
+
+For early access, Annotation caching is enabled when the product edition is `EARLY_ACCESS`.
+
+For other product editions, annotation caching is disabled, but may be forced on by setting java custom property "anno.beta" to "true".
+
+When the product edition is `EARLY_ACCESS`, the caching function may be forced off by setting java custom property `anno.beta` to `false`.
+
+The new scan engine may be enabled with cache function turned off by setting java custom property `anno.cache.disabled` to `true`.
+
+The new scan engine may be prevented from writing new data by setting java custom property `anno.cache.readOnly` to `true`.  Already generated cache data will still be read.  This may be necessary in
+certain deployment pipelines.
+
+When enabled, application class scan steps automatically write (cache) scan data, and automatically read cachedscan data.  Cache data is automatically regenerated when changes are detected in application classes.  No
+additional settings are necessary to use the new function.  (Pipeline developers, however, see "Important Note", below.)
+
+Cache data is visible as a new, nested directory of server workarea named `anno`.  (Cache data uses a binary
```

Comment:
Remove the parentheses and just leave it as a sentence (no need to be in parentheses).